src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -6093,6 +6093,13 @@ checkNullable:
             End If
         End Sub
 
+        Friend Shared Function CheckFeatures(feature As Feature, opts As VisualBasicParseOptions) As Boolean
```

Comment:
@AlekseyTs 
Let's leave this check as is, till the issue #13228 is resolved, I can implement the resolution.